tests/performance/avg_weighted.xml

```diff
@@ -32,5 +32,21 @@
     <query>SELECT avgWeighted(num_u, num)   FROM perf_avg FORMAT Null</query>
     <query>SELECT avgWeighted(num_u, num_u) FROM perf_avg FORMAT Null</query>
 
+    <query>SELECT avgWeighted(num_f,                num_f)             FROM perf_avg FORMAT Null</query>
```

Comment:
changes in this pr are not particularly coupled with `avgWeighted`, so add pls a few tests with other aggregate functions

Reply:
The only agreggate function with multiple arguments (-if argument doesn't count) that I've seen implementing addMany* with a perf improvement was avgWeighted.

ArgMin and ArgMax will also benefit once #40633 is merged and I hope that in the future more aggregations (with one or more args) will implement custom batch functions since a lot of them can benefit from it in one way or the other.